### DISCUSSION TITLE
Highlighting text from pdf for quality control

### DISCUSSION THREAD

**Discussion by TheEyesChico** [jsvine/pdfplumber#487] [Ask for help with specific PDFs]
Hi guys, your utility pdfplumber has been really helpful in processing our document in construction business. 

The challenge ahead of us now is with so much data inside pdf (can go upto 2000 pages), whatever lines we are extracting from pdf, we need to highlight the same text inside it. I looked up online and there are other modules doing this, however couldn't find any function in pdfplumber (perhaps, got overlooked, maybe you can direct me to that). 

For example in the file below, after doing automation, we want to get **Section 1.10 A.** which is under warranty. We have with us the whole line "**A. Refer to Division 01 for general warranty requirements.**" and was wondering if there was a way to highlight it just exactly below. Perhaps, a function which would take up a sentence, find it in pdf, and then highlight it (whatever the process may be).

![attachment](https://user-images.githubusercontent.com/64800839/127994243-4f72ed2c-a540-486a-9978-a5d31b5aed6f.PNG)

Looking forward to hearing from you guys!

**Comment by samkit-jain** [comment]
Hi @TheEyesChico Appreciate your interest in the library. pdfplumber only performs read operations on a PDF and for write operations, you would need to use other libraries.
